### PR TITLE
Use `github.com/pkg/errors`

### PR DESCRIPTION
**Pull request by Random-Liu**
Fixes https://github.com/containerd/cri/issues/553.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: pkg/server/sandbox_stop.go [hunk 1]
```diff
@@ -54,19 +54,19 @@ func (c *criContainerdService) StopPodSandbox(ctx context.Context, r *runtime.St
...
 	if sandbox.NetNSPath != "" && sandbox.NetNS != nil {
 		if _, err := os.Stat(sandbox.NetNSPath); err != nil {
 			if !os.IsNotExist(err) {
-				return nil, fmt.Errorf("failed to stat network namespace path %s :%v", sandbox.NetNSPath, err)
+				return nil, errors.Wrapf(err, "failed to stat network namespace path %s", sandbox.NetNSPath)
 			}
 		} else {
 			if teardownErr := c.teardownPod(id, sandbox.NetNSPath, sandbox.Config); teardownErr != nil {
-				return nil, fmt.Errorf("failed to destroy network for sandbox %q: %v", id, teardownErr)
+				return nil, errors.Wrapf(teardownErr, "failed to destroy network for sandbox %q", id)
```
Comment (mikebrow) [inline pkg/server/sandbox_stop.go]: impressed you resisted changing testdownErr -> err :-)

**Comment by yanxuean** [comment]
Do the containerd not allow to use fmt?

**Comment by thaJeztah** [comment]
> Do the containerd not allow to use fmt?

`errors.Wrap()` preserves the orginal error, whereas `fmt.Errorf()` does not, so its generally more useful for use wrap

**Comment by yanxuean** [comment]
Thanks @thaJeztah

**Pull request merged**